<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseEditor</class>
 <widget class="QDialog" name="CourseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>18</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>72</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Course ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>126</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Semester</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Coefficient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LectureButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>290</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lecture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LabButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TutorialButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CourseNameEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>18</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CourseIDedit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>72</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Semesteredit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>126</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Coefedit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteCourseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources for SMS/removeitem_resized.jpeg);
background-position: center;
background-repitition: no-repitition;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SessionActivation">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Sessions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Teachers :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="TeacherListWidget">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>290</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTeacherToCourseButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(180, 180, 180);
background-image: url(:/resources for SMS/Plus2.jpeg);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveTeacherFromCourse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources for SMS/Minus.jpeg);
background-position: center;
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>LectureButton</zorder>
  <zorder>LabButton</zorder>
  <zorder>TutorialButton</zorder>
  <zorder>CourseNameEdit</zorder>
  <zorder>CourseIDedit</zorder>
  <zorder>Semesteredit</zorder>
  <zorder>Coefedit</zorder>
  <zorder>DeleteCourseButton</zorder>
  <zorder>SessionActivation</zorder>
  <zorder>SaveButton</zorder>
  <zorder>label_6</zorder>
  <zorder>TeacherListWidget</zorder>
  <zorder>AddTeacherToCourseButton</zorder>
  <zorder>RemoveTeacherFromCourse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
